### ISSUE TITLE
A11y_AzureDataStudio_ClusterSettings_ScreenReader:NVDA does not announces the word "Required" for the mandatory filed.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/azuredatastudio#7592]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:** #A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#DesktopApp;#NVDA;#JAWS;#Win10;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727ff9fe68421b004bf270cd509e4c5de165
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
NVDA Version:2019.2

**Repro Steps:**

1. Install /Launch the application Azure data studio.
2. Navigate to connections and click on more action button.
3. Select “Deploy SQL Server big data cluster” from more action drop down.
4. Click on “Deploy SQL Server big data cluster”.
5. Check the check box I accept in the “select the deployment option” screen and click on
select button.
6. Select any block and click on next button and again click on next button.
7. " Cluster settings" screen is opened
8. Start the NVDA and navigate to mandatory fields using tab navigation
9. Check the the NVDA screen reader announces the word "Required" for the mandatory fields
or not

**Actual:**
NVDA does not announces the word "Required" for the mandatory fields.

**Expected:**
NVDA should announce the word "Required" for the mandatory fields.

**Same issue observed in JAWS.**

**same issue observed for the below scenario:**
**1. services sittings**
1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button.
8.	Verify that all the elements of " Cluster settings" are accessible and MAS Compliant.

**User Impact:**
Screen reader user will not understand the field is mandatory need to fill the data. If the screen reader does not announces required.

**Attachment for Reference:**
[7592_A11y_AzureDataStudio_ClusterSettings_ScreenReader](https://github.com/microsoft/azuredatastudio/files/3712291/7592_A11y_AzureDataStudio_ClusterSettings_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-May2019;
